### DISCUSSION TITLE
2.0.0发版本发现告警管理组没有关联用户的按钮？

### DISCUSSION THREAD

**Discussion by wmky** [apache/dolphinscheduler#7296] [Q&A]
2.0.0版本发现告警管理组没有关联用户的按钮？请官方同学帮忙解答下，谢谢！

**Comment by wmky** [comment]
感谢社区罗小喵的帮助。经过询问社区得到答复是2.0.0版本做了一些改版，告警组绑定用户的功能已经放在了创建告警实例中添加用户了。总体发现这样的改动不利益随时变更告警组与人员关系问题。

**Reply by RedemptionC** [reply]
maybe you can make this as answer
